lib/google/gax/api_callable.rb

```diff
@@ -33,45 +33,54 @@
 
 module Google
   module Gax
-    # Converts an rpc call into an API call governed by the settings.
+    # Converts an RPC call into an API call.
     #
-    # In typical usage, +func+ will be a proc used to make an rpc request.
-    # This will mostly likely be a bound method from a request stub used to make
-    # an rpc call.
+    # In typical usage, +func+ will be a proc used to make an RPC request.
+    # This will mostly likely be a bound method from a request Stub used to make
+    # an RPC call.
     #
     # The result is created by applying a series of function decorators
-    # defined in this module to +func+.  +settings+ is used to determine
-    # which function decorators to apply.
+    # defined in this module to +func+.
     #
-    # The result is another proc which for most values of +settings+ has the
-    # same signature as the original.
+    # The result is another proc which has the same signature as the original.
     #
     # @param func [Proc] used to make a bare rpc call
-    # @param settings [CallSettings] provides the settings for this call
+    # @param timeout [Numeric] client-side timeout for API calls
+    # @param metadata [Hash]  request headers
     # @param params_extractor [Proc] extracts routing header params from the
     #   request
     # @param exception_transformer [Proc] if an API exception occurs this
     #   transformer is given the original exception for custom processing
     #   instead of raising the error directly
-    # @return [Proc] a bound method on a request stub used to make an rpc call
-    def create_api_call(func, settings, params_extractor: nil,
-                        exception_transformer: nil)
-      api_caller = proc do |api_call, request, _settings, block|
-        api_call.call(request, block)
-      end
+    # @return [Proc] A bound method on a request stub used to make an rpc call.
+    #   The argumentd for the bound method are:
+    #
+    #   * request - Request object
+    #   * options - CallOptions object
+    #   * block named argument - Proc object for yielding the operation
+    def create_api_call(func, timeout: 60, metadata: {},
```

Comment:
Instead of passing a CallSettings object, the values are passed in individually. This means that if a value is not passed in, a default value will be used instead.